assets/styles/components/_hint.scss

```diff
@@ -1 +1,13 @@
 @import 'govuk-frontend/components/hint/hint';
+
+.govuk-hint {
+  ul, ol {
+    margin-left: $govuk-gutter-half;
+
+    li {
+      text-indent: -26px;
+      padding-left: 26px;
+      margin-bottom: 10px;
+    }
+  }
+}
```

Comment:
these seem like "normal" lists? Why are we only applying these styles to lists in hints?

Reply:
It adds the extra indentation that Affy wanted.